test/plug/router_test.exs

```diff
@@ -178,6 +181,12 @@ defmodule Plug.RouterTest do
     assert conn.resp_body == "root"
   end
 
+  test "dispatch empty body plug" do
+    assert_raise RuntimeError, fn ->
```

Comment:
I'm not sure what the error message is here, and for this reason maybe it's a good idea to assert on it as well with something like:

```elixir
assert_raise RuntimError, "this and that", fn -> ... end
```

(you probably know but you can use a regex to assert on the error message, it doesn't have to be a string 😃)